niceman/distributions/debian.py

```diff
@@ -179,6 +194,30 @@ def normalize(self):
         #   would make us require supporting flexible typing -- string or a list
         pass
 
+    def satisfies_package(self, package):
+        """return True if this distribution (self) satisfies the requirements 
+        of the passed package"""
+        if not isinstance(package, Package):
+            raise TypeError('satisfies_package() requires a package argument')
+        if not isinstance(package, DEBPackage):
+            return False
+        for p in self.packages:
+            if p.satisfies(package):
+                return True
+        return False
```

Comment:
see comment below ... eventually we might want to make it more efficient/avoid full list lookup every time.